README.md

```diff
@@ -162,12 +162,40 @@ To delete a cluster, run:
 eksctl delete cluster --name=<name> [--region=<region>]
 ```
 
+### Creating nodegroup
+
+You can add one ore more nodegroups in addition to the initial nodegroup created along with the cluster.
+
+To create an additional nodegroup, run:
+
+```
+eksctl create nodegroup --cluster=<cluser name>
+```
+
+### Listing nodegroups
+
+To list the details about a nodegroup or all of the nodegroups, use:
+
+```
+eksctl get nodegroup --cluster=<cluster name> [<nodegroup SEQ>]
```

Comment:
I actually don't know! Perhaps we'd better ask @richardcase as he seems to have decided to use SEQ.

I'm personally fine with changing it to use auto-generated names instead. https://github.com/weaveworks/eksctl/pull/281#discussion_r238560912

 If I only I can address it in an another PR so that this changes between @richardcase's work and mine become clearer, which may make his review easier.

Reply:
Let's wait for @richardcase to share his opinion on that then, I'd love to have this from the beginning, but incrementally is also fine.

Reply:
Not sure i entirely agree that names are any less error prone. However, it could be easier when identifying specific nodegroups (i.e. a GPU optimized enabled group) but that can be done via labels. 

Perhaps we can have both? We always have a sequence number as that denotes the sequence in which they where created and its easy to to just increment the sequence to make a unique group name. We could then have a name flag that if specified will result in a specific label being attched to the stack that we could then pull out when we do the `get`. WDYT?

Reply:
That's pretty much what I meant even though I was super brief 😄 A user provided name is the best we can do and it's what tools like kops are already providing. To give a user perspective, I use this all the time for example to do canary node groups to test new AMIs, would be great to have that!
I'm pretty much agree with what @mumoshu said here :-)

Reply:
Yay 😄

I'm prepared to submit an another PR to change the SEQ number to specific/auto-generated names if we could build a consensus with @richardcase and @errordeveloper.